zilencer/management/commands/populate_db.py

```diff
@@ -1087,7 +1089,7 @@ def generate_and_send_messages(
             message.subject = random.choice(possible_topics[message.recipient.id])
             saved_data["subject"] = message.subject
 
-        message.date_sent = choose_date_sent(num_messages, tot_messages, options["threads"])
+        message.date_sent = choose_date_sent(num_messages, tot_messages, starting_n_days_ago, options["threads"])
```

Comment:
What's the `manage.py` command that calls this code? I imagine `choose_date_sent` isn't being called directly from the command line. Do I need to pipe this argument through from the `add_arguments` for `populate_db` and pass it through in `generate_and_send_messages`?